docs/migrating-from-0.2-to-1.0.md

```diff
@@ -363,12 +365,68 @@ The `embedded-hal` project now spans several crates, where some functionality ha
 
 | Crate | crates.io | Docs | |
```

Comment:
maybe it makes sense to also add a comment explicitly stating that these crates have individual release cycles and not all of them are hitting v1.0 at the same time?

Reply:
done